apps/rule-manager/conf/routes

```diff
@@ -3,49 +3,54 @@
 # https://www.playframework.com/documentation/latest/ScalaRouting
 # ~~~~
 
-GET     /                           controllers.HomeController.index()
-GET     /oauthCallback              controllers.HomeController.oauthCallback()
-# A temporary endpoint to test permissions
-GET     /editPermissionCheck        controllers.HomeController.editPermissionCheck()
+GET     /                               controllers.HomeController.index(path = "")
+GET     /oauthCallback                  controllers.HomeController.oauthCallback()
 
-+anyhost
-GET     /healthcheck                controllers.HomeController.healthcheck()
++anyhostå
+GET     /api/healthcheck                controllers.HomeController.healthcheck()
```

Comment:
The default healthcheck endpoint is /healthcheck (https://github.com/guardian/typerighter/blob/main/cdk/lib/__snapshots__/index.test.ts.snap#L1515) - so unless we remove the `api` prefix here, or adjust the healthcheck endpoint in cdk, I think we'll see a failed deployment (as the instance will be marked as unhealthy).

Reply:
Ah good point, there's no reason that can't be a reserved endpoint like `oauthCallback` - will update now.